### PR TITLE
WIP: DOC-362 / CMS-252 - Migrate External Drupal 9 Site to Pantheon, edits

### PR DESCRIPTION
**Pull request by namespacebrian**
Closes #

## Summary

**[Doc Page Title](https://pantheon.io/docs/doc-title)** -  <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here

--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6682](https://pr-6682--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `55.00000000000001` | `-5` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6682` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/30140/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/30140/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6682--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md [hunk 1]
```diff
@@ -15,6 +15,24 @@ editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
 
 In this doc, you'll migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with Integrated Composer on Pantheon.
 
+## Unresolved Questions / TO-DOs
+
+- [How] Do we address nested docroot vs not-nested docroot?
```
Comment (greg-1-anderson) [inline source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md]: Composer-managed sites should almost always have a nested docroot. For now we could just make that a requirement. We should address the fact that the docroot name might be different on the source site. It's best to rename it to `web`, as Pantheon expects.

**Comment by EdwardAngert** [comment]
for me: squash and merge so the wip commits aren't in history

File: source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md (hunk 1 above)
Comment (namespacebrian) [reply source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md]: Okay, note @EdwardAngert Greg thinks we should address the subject of whether the existing site had a nested docroot and what it was named.

@greg-1-anderson in my draft, I had settled on simply not mentioning it and referring to the old/existing site paths relative to the docroot

**Comment by namespacebrian** [comment]
Here's the screenshots I took of the "Create new site" process, in case they're useful, but I'm guessing docs already has screenshots of that process somewhere...

create-d9-site1
create-d9-site2
create-d9-site3
create-d9-site4

**Pull request merged**